src/main/java/homework_2/traffic_light/TrafficLight.java

```diff
@@ -0,0 +1,58 @@
+package homework_2.traffic_light;
+
+import java.util.Scanner;
+
+public class TrafficLight {
+
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+
+    private static void trafficLight(String inputStr) {
```

Comment:
you can use it as ExtraTrafficLight.
default TrafficLight realization should provides seconds gone from the day start as input parameter.

Reply:
you're right, I thought we could make only program with hh:mm:ss format.
let me add new app!